### ISSUE TITLE
why is lr so low?

### ISSUE THREAD

**Issue by cjt222** [eragonruan/text-detection-ctpn#55]
hello,i have a question, i try ro train model and if i set lr 0.00001, it will converge slowly,but if i set lr higher, it will never converge and even nan, in fact, i feel that 0.00001 is so low that model can learning nothing, why does it happen?

**Comment by cjt222** [comment]
i also have a question, i replace VGG with Mobilenet, but it has trouble performence， i want to know if some parameters i may need to change.if i combine low level and high level , do i need to change anchor scale?in fact, their receptive field is different,thanks for your answer

**Comment by eragonruan** [comment]
@cjt222   
- how to set learning rate depends on your task and the solver. 
- your may refer to the paper to decide how to set the parameters
- the anchor scale is relative to the origin image, hence I don't think you have to modify it
